deploy/cephcsi/image/Dockerfile

```diff
@@ -1,15 +1,51 @@
-FROM ceph/ceph:v15
-LABEL maintainers="Ceph-CSI Authors"
-LABEL description="Ceph-CSI Plugin"
+ARG SRC_DIR="/go/src/github.com/ceph/ceph-csi/"
+ARG GO_ARCH
+ARG BASE_IMAGE=ceph/ceph:v15
 
-# To support cross building, do NOT RUN native binaries here.
-# If we have to run native binaries, qemu-user-static must be installed on
-# build host and mounted to container.
+FROM ${BASE_IMAGE} as builder
 
-# Removing ceph-iscsi repo to workaround the repo issue while upgrading
-#RUN rm -f /etc/yum.repos.d/ceph-iscsi.repo && yum -y update && yum clean all
-ENV CSIBIN=/usr/local/bin/cephcsi
+LABEL stage="build"
 
-COPY cephcsi $CSIBIN
+ARG GOLANG_VERSION=1.13.9
+ARG CSI_IMAGE_NAME=quay.io/cephcsi/cephcsi
+ARG CSI_IMAGE_VERSION=canary
+ARG GO_ARCH
+ARG SRC_DIR
+ARG GIT_COMMIT
+ARG GOROOT=/usr/local/go
+RUN dnf install libcephfs-devel librados-devel librbd-devel /usr/bin/cc /usr/bin/c++ make -y
+
+RUN mkdir -p /usr/local/go && \
+    curl https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-${GO_ARCH}.tar.gz | tar xzf - -C /usr/local/go --strip-components=1
+
+
+ENV GOROOT=$GOROOT \
+    GOPATH=/go \
+    CGO_ENABLED=1\
```

Comment:
Isn't this enabled by default? Missing a space before the `\`. Also on the following lines.